src/test/snapshots/client.test.ts.md

```diff
@@ -10,6 +10,7 @@ Generated by [AVA](https://avajs.dev).
 
     {
       enabled: true,
+      featureEnabled: true,
```

Comment:
there was a value `enabled` previously. Doesn't it correspond to the feature being enabled? I don't think we have a variant enabled/disabled status

Reply:
If that's the case I could massively simplify the fix

Reply:
the enabled field is there but I thought it's for the variant not the toggle status